prov/efa/test/README.md

```diff
@@ -0,0 +1,54 @@
+# EFA unit tests
+
+## How to run
+
+To run efa unit tests, you will need to have cmocka installed.
+Cmocka Mirror: https://cmocka.org/files/
+Install Instructions: https://gitlab.com/cmocka/cmocka/-/blob/master/INSTALL.md
+
+You will need to configure libfabric with ```--with-cmocka=<path_to_cmocka_install>```.
```

Comment:
As I mentioned before, `with-cmocka -> with-efa-unit-test`

Reply:
switching back to `--enable-efa-unit-test=DIR`